src/main/java/de/retest/recheck/ui/descriptors/idproviders/AbstractFirstNonNullRetestIdProvider.java

```diff
@@ -21,7 +20,7 @@ public String getRetestId( final IdentifyingAttributes identifyingAttributes ) {
 		final String htmlId = normalizeAndCut( identifyingAttributes.get( "id" ) );
 		final String name = normalizeAndCut( identifyingAttributes.get( "name" ) );
 		final String type = normalizeAndCut( cutTypeQualifier( identifyingAttributes.get( "type" ) ) );
-		final String id = returnFirstNonBlank( text, htmlId, name, type, randomUUID().toString() );
+		final String id = returnFirstNonBlank( text, htmlId, name, type );
```

Comment:
If untested, please add a test. Otherwise, this may be the reason for the failing tests (I haven't looked into them): https://travis-ci.com/github/retest/recheck/builds/154251999

Reply:
Sorry ... this doesn't belong to this branch...